ShestakUI/Modules/Cooldowns/EnemyCD.lua

```diff
@@ -102,13 +102,17 @@ local CreateIcon = function()
 	return icon
 end
 
-local StartTimer = function(name, sID)
+local StartTimer = function(sGUID, sID)
 	local _, _, texture = GetSpellInfo(sID)
 	local icon = CreateIcon()
 	icon.Texture:SetTexture(texture)
 	icon.Texture:SetTexCoord(0.1, 0.9, 0.1, 0.9)
 	icon.endTime = GetTime() + T.EnemySpells[sID]
-	local color = (CUSTOM_CLASS_COLORS or RAID_CLASS_COLORS)[select(2, UnitClass(name))]
+	local locClass, _, _, _, _, name, _ = GetPlayerInfoByGUID(sGUID)
```

Comment:
Should be without , _ in the end
local locClass, _, _, _, _, name = GetPlayerInfoByGUID(sGUID)

Reply:
The other informations gather by GetPlayerInfoByGUID are not interesting